packages/api/src/schema/schema.yaml

```
@@ -239,6 +241,42 @@ components:
           example: D8321C3E-B29C-45EB-A1BB-A623D8BE0F65
           description:
             ID of object store where to which this stream was recorded
+        pushTargets:
+          type: array
+          example: [{ "id": "PUSH123", "profile": "720p" }]
+          description:
+            References to targets where this stream will be simulatenously
+            pushed to
+          items:
+            type: object
+            required: [profile]
+            additionalProperties: false
+            properties:
+              profile:
+                type: string
+                description:
+                  Name of transcoding profile that should be sent. Use "source"
+                  for pushing source stream data
+                minLength: 1
+                maxLength: 500
+                example: aac
```

"aac" would be a weird profile for this, implies an audio-only profile that we don't support. Alternative:
```suggestion
                example: 720p
```

---

Oh good catch. I had copied it from the `profile.name` field in this schema, soo I'll update that one too.